tests/acceptance/30_generic_methods/file_remove.cf

```diff
@@ -46,14 +49,26 @@ bundle agent check
 {
   classes:
     "file_exists"      expression => fileexists("${init.destination_file}");
-
-    "ok"                   expression => "!file_exists.file_remove_${init.destination_file_canon}_repaired.!file_remove_${init.destination_file_canon}_error";
+    "ok_1"             expression => "!file_exists.file_remove_${init.destination_file_canon}_repaired.!file_remove_${init.destination_file_canon}_error";
+    "ok_2"             expression => "file_remove_${init.not_a_file_canon}_kept.!file_remove_${init.not_a_file_canon}_repaired.!file_remove_${init.not_a_file_canon}_error";
+    "ok"                      and => { "ok_1", "ok_2" };
 
   reports:
+    file_exists::
+      "First test: File ${init.destination_file} exists even though it should have been removed!";
+    ok_1::  "First test OK (removing existing file, ${init.destination_file})";
+    !ok_1:: "First test failed (removing existing file, ${init.destination_file})";
+    ok_2::  "Second test OK (removing a non existant file, ${init.not_a_file_canon})";
+    !ok_2:: "Second test failed (removing a non existant file, ${init.not_a_file_canon})";
```

Comment:
Other tests just output PASS or FAIL, won't this additional debug output cripple the results ?